brainstormforce/astra
# wip: feat: Content & Title meta fields added in Blog & Single Posts structures

### Description
- New Title & Content draggable fields added in case of Blog Archive & Single Posts customizer

### Screenshots


### Types of changes
- Added new draggable fields in structure
- Maintained backward compatibilities

### How has this been tested?
- Check backward compatibility
- Check for new users case
- Check by using following action in prev version and verification the same in this release (on Blog & Single Post pages)
1. astra_entry_content_before
2. astra_entry_content_after

### Checklist:
- My code is tested
- My code passes the PHPCS tests
- My code follows accessibility standards
- My code has proper inline documentation 
- I've included any necessary tests
- I've added proper labels to this pull request
